packages/design-system/package.json

```diff
@@ -45,7 +45,6 @@
     "@types/react-modal": "^3.13.1",
     "@types/styled-components": "^5.1.26",
     "big.js": "^6.2.1",
-    "focus-visible": "^5.2.0",
```

Comment:
This was a polyfill correct?  Just curious if there was a browser version we were waiting on for support or if this is cleanup.

Reply:
Yes, see PR description.